Jenkinsfile

```diff
@@ -1,7 +1,7 @@
 /*
  * Copyright:: Copyright (c) 2017-present Sonatype, Inc. Apache License, Version 2.0.
  */
-@Library('zion-pipeline-library')
+@Library('ci-pipeline-library')
```

Comment:
This works, but private-pipeline-library might be better.  private-pipeline-library is the global, trusted reference to https://github.com/sonatype/ci-pipeline-library

Reply:
Thanks @tneer!

Reply:
45cab30